Apps/Common/src/MapProfiles/UserProfileProfile.cs

```diff
@@ -39,6 +40,11 @@ public UserProfileProfile()
             this.CreateMap<UserProfile, SupportUser>()
                 .ForMember(dest => dest.Hdid, opt => opt.MapFrom(src => src.HdId))
                 .ReverseMap();
+
+            this.CreateMap<ResourceDelegate, SupportUser>()
+                .ForMember(d => d.Hdid, opts => opts.MapFrom(s => s.ResourceOwnerHdid))
```

Comment:
The ResourceOwnerHdid is for the dependent; the ProfileHdid is for the delegate.